playerbot/strategy/values/MoveStyleValue.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "../../LootObjectStack.h"
+#include "../Value.h"
+#include "SubStrategyValue.h"
+
+namespace ai
+{
+    class MoveStyleValue : public SubStrategyValue
+	{
+	public:
+        MoveStyleValue(PlayerbotAI* ai, string defaultValue = "", string name = "move style", string allowedValues = "wait,noedge") : SubStrategyValue(ai, defaultValue, name, allowedValues) {}
```

Comment:
`MoveStyleValue(PlayerbotAI* ai, const string& defaultValue = "", const string& name = "move style", const string& allowedValues = "wait,noedge") : SubStrategyValue(ai, defaultValue, name, allowedValues) {}` to prevent copying